react-site/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@patternfly/react-core": "^3.75.2",
-    "@patternfly/react-icons": "^3.10.14",
+    "@patternfly/react-core": "4.18.5",
```

Comment:
It's better to use semantic versioning in here with the ^ and then check in the lock file.